python/ray/experimental/serve/utils.py

```diff
@@ -15,7 +15,8 @@ def parse_request_item(request_item):
     if request_item.request_context == TaskContext.Web:
         is_web_context = True
         asgi_scope, body_bytes = request_item.request_args
-        flask_request = build_flask_request(asgi_scope, io.BytesIO(body_bytes))
+        flask_request = build_flask_request(asgi_scope,
+                                            io.BytesIO(body_bytes[0]))
```

Comment:
why `[0]`?

Reply:
Try running this code - 
```python
import ray
@ray.remote
def f(a):
    return a
ray.init()
result = ray.get(f.remote(b""))
print(result)
```

Reply:
This is a bug, can you file a github issue with traceback?